pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -113,6 +113,38 @@ const (
 	AnnotationFalse    = "false"
 )
 
+var ValidKumaAnnotations = map[string]bool{
```

Comment:
Kind of a nit but IMO this should be `map[string]struct{}` unless there is a semantic difference between not being in the map and the value being `false`.

Could we maybe introduce a method like `registerKumaAnnotation` that we call to create an annotation?

```
func registerKumaAnnotation(a string) string {
  ValidKumaAnnotations[a] = struct{}{}
  return a
}
```

And then:

```
var (
  KumaSidecarInjectedAnnotation = registerKumaAnnotation("kuma.io/sidecar-injected")
)
```